templates/typescript_gapic/_util.njk

```diff
@@ -262,7 +262,7 @@ request.{{ oneComment.paramName.toCamelCase() }}
     let routingParameter = {};
 {%- for paramArray in method.dynamicRoutingRequestParams %}
   {%- for sameParamArray in paramArray %}
```

Comment:
nit: could we rename `sameParamArray` as `param` for simplicity. go/tott/431 is good reference.

Reply:
Makes sense - thanks for the reference!